modules/CmpdReg/src/client/templates/AppControllerView.inc

```diff
@@ -31,7 +31,7 @@
 
 	</div>
 
-	<div id="footer"><p>&#169; 2011-2015 John McNeil & Co., Inc.</p></div>
+	<div id="footer"><p>&#169; 2011-2017 John McNeil & Co., Inc.</p></div>
```

Comment:
Should we go ahead and bump to 2018? @jmcneil86043 @bffrost

Reply:
I defer to John on this one